src/main/java/org/pkcs11/jacknji11/CKM.java

```diff
@@ -21,11 +21,13 @@
 
 package org.pkcs11.jacknji11;
 
+import com.sun.jna.Memory;
+import com.sun.jna.Pointer;
+
 import java.util.HashMap;
 import java.util.Map;
 
-import com.sun.jna.Memory;
-import com.sun.jna.Pointer;
+import static org.pkcs11.jacknji11.Hex.b2s;
```

Comment:
nit: I prefer to avoid churn like changing order of imports and adding extra static imports.

Reply:
done

Reply:
Remove static import of Hex.b2s and use Hex.b2s rather than b2s below

Reply:
done